lib/editor/util/validation.js

```diff
@@ -236,27 +246,30 @@ export function validate (
     case 'LATITUDE':
       const isNotLat = value > 90 || value < -90
       if (isNotLat) {
-        return validationIssue('Field must be valid latitude.')
+        return validationIssue('Field must be valid latitude.', field.name)
       }
       if (isOptionalAndEmpty && locationType !== null && (typeof locationType === 'number' && locationType <= 2)) {
-        return validationIssue('Latitude and Longitude are required for your current location type')
+        return validationIssue('Latitude and Longitude are required for your current location type', field.name)
       }
       return false
     case 'LONGITUDE':
       const isNotLng = value > 180 || value < -180
       if (isNotLng) {
-        return validationIssue('Field must be valid longitude.')
+        return validationIssue('Field must be valid longitude.', field.name)
       }
       if (isOptionalAndEmpty && typeof locationType === 'number' && locationType <= 2) {
-        return validationIssue('Latitude and Longitude are required for your current location type')
+        return validationIssue('Latitude and Longitude are required for your current location type', field.name)
       }
       return false
     case 'TIME':
       const isValid = moment(value, 'HH:mm:ss', true).isValid()
       if (isRequiredButEmpty) {
         return emptyFieldValidationIssue()
       } else if (!valueDoesNotExist && !isValid) {
-        return validationIssue('Field must be valid time')
+        return validationIssue('Field must be valid time', field.name)
+      // $FlowFixMe: Flow doesn't recognize the BookingRule check.
+      } else if (entity && entity.booking_type) {
```

Comment:
You'll need to add `GtfsBookingRule` to the possible types under `Entity`, I think.

Reply:
Yep, that resolves these errors but seems to uncover some other issues? Any idea how to resolve them? They're in the latest commit

Reply:
They seem to be unrelated to the addition of GtfsBookingRule to `Entity` which is why I'm confused.

Reply:
See #978 for adjustments to make type check pass.